packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries/BinaryOptionLongShortPairFinancialProductLibrary.sol

```diff
@@ -3,49 +3,43 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/utils/math/SafeMath.sol";
 
-import "./ContractForDifferenceFinancialProductLibrary.sol";
+import "./LongShortPairFinancialProductLibrary.sol";
 import "../../../../common/implementation/Lockable.sol";
 
 /**
- * @title Binary Option Contract For Difference Financial Product Library.
- * @notice Adds settlement logic to binary option CFDs. Binary options settle with all collateral allocated to
+ * @title Binary Option Long Short Pair Financial Product Library.
+ * @notice Adds settlement logic to binary option LSPs. Binary options settle with all collateral allocated to
  * either the long or short side, depending on the settlement price. They can be used to make prediction markets or any
  * kind of binary bet. Settlement is defined using a strike price which informs which side of the bet was correct. If
  * settlement price is greater or equal to the strike then all value is sent to the long side. Otherwise, all value
  * is sent to the short side. The settlement price could be a scalar (like the price of ETH) or a binary bet with
  * settlement being 0 or 1 depending on the outcome.
  */
-contract BinaryOptionContractForDifferenceFinancialProductLibrary is
-    ContractForDifferenceFinancialProductLibrary,
-    Lockable
-{
+contract BinaryOptionLongShortPairFinancialProductLibrary is LongShortPairFinancialProductLibrary, Lockable {
     using FixedPoint for FixedPoint.Unsigned;
     using SafeMath for uint256;
 
-    struct binaryContractForDifferenceParameters {
+    struct binarylongShortPairParameters {
```

Comment:
nit(unrelated to this PR): shouldn't type names be capitalized?

`BinaryLongShortPairParameters`

Reply:
it should! good call out